en/includes/guides/organization-management/email-domain-based-organization-discovery.md

```diff
@@ -1,193 +1,212 @@
-One way to achieve this is through organization discovery based on criteria such as email domain and user attributes.
-{{ product_name }} offers a solution for email domain-based organization discovery.
+In the realm of Business-to-Business (B2B), seamlessly authenticating users from external organizations into your applications is essential for a smooth user experience. One way to achieve this is through email domain-based organization discovery. With this feature, your application is able to provide a customized and a familiar login experience for a user based on the domain of their email address.
 
-The primary purpose of email domain based discovery is to route B2B users from various organizations to their respective authentication options based on their email addresses.
+For example, when users log in with an `@xyz.com` email address, they will be automatically redirected to the login screen customized for the `XYZ` organization, providing a familiar and consistent experience.
 
-This guide explains how to configure and use email domain-based organization discovery in {{ product_name }}.
+This guide explains how you may use {{product_name}} to configure email domain-based organization discovery for your B2B applications.
 
 ## Scenario
 
-Consider **Guardio Insurance** is a B2B SaaS provider and `Best Car Mart` and `Glory Car Mart` are two of its customer/partner organizations.
-Guardio Insurance wants to resolve the organization of a user based on the email domain of the user's email address and route them to the respective authentication options.
+**Guardio Insurance** is an insurance provider. *Best Car Mart* and *Glory Car Mart* are two of its customer organizations. When a user enters an email address to log into the Guardio Insurance platform, it is required to resolve a user's organization based on the email domain and redirect them to the respective organization's login screen.
+
 
 ![Email domain based organization discovery scenario]({{base_path}}/assets/img/guides/organization/manage-organizations/email-domain-based-organization-discovery.png){: width="700" style="display: block; margin: 0;"}
 
-User Alice with email `alice@bcmart.com` should be routed to the authentication options provided by the `Best Car Mart` organization,
-and user Bob and Ben, with emails `bob@gcmart.com` and `ben@glory.com` should be routed to the authentication options provided by the `Glory Car Mart` organization.
+In this example scenario,
+
+- User Alice with email `alice@bcmart.com` should be routed to the login screen of `Best Car Mart` organization.
+- User Bob and Ben, with emails `bob@gcmart.com` and `ben@glory.com` should be routed to the login screen of `Glory Car Mart` organization.
 
 ## Enable email domain based organization discovery
 
-!!! note
-    - This feature can only be used when email address is configured as the username.
```

Comment:
I think it's better to keep this here itself instead of moving to before you begin section since this restriction applies to the whole feature

Reply:
fixed with 3033eb4d1d38180c4ca1d0ad42b202f75b46363f